src/ducks/apps/index.js

```diff
@@ -183,7 +183,7 @@ export function fetchRegistryApps (lang = 'en') {
       return Promise.all(apps.map(app => {
         const appName = (app.name && (app.name[lang] || app.name.en)) || app.slug
         const appDesc = (app.description && (app.description[lang] || app.description.en)) || ''
-        return _getIcon(app.logo_url)
+        return _getIcon(`/registry/${app.slug}/icon`)
```

Comment:
Does it mean that we are still relying on a local icon ?

Reply:
Nope it's an icon provided by cozy-stack using this route. But we still have a local fallback icon in case of errors when fetching the app icon.